From fad4079a7376843f6b6b7f357663cecd2b96b1f6 Mon Sep 17 00:00:00 2001
From: Candice Li <candice.li@amd.com>
Date: Wed, 10 Jul 2019 19:12:58 +0800
Subject: [PATCH 2847/2940] Revert "drm/amdgpu: Disable ras features on all IPs
 before gpu reset"

This reverts commit 7370f3be92e2d9cd847145ea402672fe9f51664c.

Change-Id: I783d89bc8bb587d3045ade3a0c4ad8e75cc0ac4d
Signed-off-by: Candice Li <candice.li@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_device.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_device.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_device.c
index 6edc9d623148..40989c0c81d4 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_device.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_device.c
@@ -3730,10 +3730,6 @@ int amdgpu_device_gpu_recover(struct amdgpu_device *adev,
 
 	/* block all schedulers and reset given job's ring */
 	list_for_each_entry(tmp_adev, device_list_handle, gmc.xgmi.head) {
-		/* disable ras on ALL IPs */
-		if (amdgpu_device_ip_need_full_reset(tmp_adev))
-			amdgpu_ras_suspend(tmp_adev);
-
 		for (i = 0; i < AMDGPU_MAX_RINGS; ++i) {
 			struct amdgpu_ring *ring = tmp_adev->rings[i];
 
-- 
2.17.1

